### ISSUE TITLE
Can't use most of the MyPaint UI after a few minutes of use.

### ISSUE THREAD

**Issue by ConsoleDevPumpkin** [mypaint/mypaint#1144]
#### Description of the problem
Yeah, so that's the problem. About 3 minutes in, I have to save the project and kill the process to get it running again.

#### Basic system details
MyPaint version: 2.0.1
Operating system: Windows 10 build 19042

#### Steps to reproduce
1. Start MyPaint
2. Use it 4 a few minutes

**Comment by jplloyd** [comment]
Some information that could be helpful:
1. Is this a consistent problem? Has it occurred more than once?
2. What input hardware (tablet, etc) are you using?
2. Which parts of the UI are still usable?
3. Does the resource manager give any indications of high CPU usage or high memory usage when/after the UI stops working?

**Comment by ConsoleDevPumpkin** [comment]
> Some information that could be helpful:
> 
> 1. Is this a consistent problem? Has it occurred more than once?
> 2. What input hardware (tablet, etc) are you using?
> 3. Which parts of the UI are still usable?
> 4. Does the resource manager give any indications of high CPU usage or high memory usage when/after the UI stops working?

Yeah, it happens every time. I can still draw and change brushes but nothing else. I'm using the Wacom intuos PTH-651, and as for the resources, it uses about 4% of the RAM and almost no CPU.

**Comment by jplloyd** [comment]
Thank you. I have not been able to reproduce this, but I did get a bug (not sure exactly how) where the visuals were offset vertically with the actual control areas, meaning that I had to click underneath the controls to activate them. I assume this is not the same issue you have?

Does keyboard inputs work in general, or just changing brushes?

I will do some more testing this weekend.

**Comment by ConsoleDevPumpkin** [comment]
> Thank you. I have not been able to reproduce this, but I did get a bug (not sure exactly how) where the visuals were offset vertically with the actual control areas, meaning that I had to click underneath the controls to activate them. I assume this is not the same issue you have?
> 
> Does keyboard input work in general, or just changing brushes (assuming that you change them using the keyboard)?
> 
> I will do some more testing this weekend.

I don't use the keyboard in the app, so I don't know if the inputs work.
